openedx/features/marketplace/decorators.py

```diff
@@ -0,0 +1,11 @@
+from django.http import HttpResponseForbidden
+
+
+def has_affiliated_user(function=None):
+    def _deco(request, *args, **kwargs):
+        if request.user.extended_profile.organization is None:
+            return HttpResponseForbidden('Access denied!')
+
+        return function(request, *args, **kwargs)
```

Comment:
Let's please simplify this little more
```
if request.user.extended_profile.organization:
	return function(request, *args, **kwargs)
return HttpResponseForbidden('Access denied!')
```

or 

```
return function(request, *args, **kwargs) if request.user.extended_profile.organization else HttpResponseForbidden('Access denied!')
```

or 

```
return (function(request, *args, **kwargs) if request.user.extended_profile.organization 
		else HttpResponseForbidden('Access denied!'))
```

Reply:
Thumbsup to the first one